tools/qr-generator/src/generator/token.js

```diff
@@ -0,0 +1,44 @@
+import { generateUniqueCodes } from './codes.js'
+import { padNumberToSameLength } from '../util.js'
+
+const getTokenName = (name, index) => name.replace('{index}', index)
+
+const generateTokenData = (i, token, args) => {
```

Comment:
A bit less lines :D

```
const generateTokenData = (i, token, args) => ({
  title: getTokenName(args.name, padNumberToSameLength(i + 1, args.count)),
  index: i,
  token,
  ...args
});
```

Reply:
The first destructuring has to stay, because `args` contains more fields than the actual token data, that we put in the CSV
So I updated the code to look like this:
```js
const generateTokenData = (i, token, args) => {
  const {
    count,
    name,
    visible,
    type,
    icon,
    rarity,
    score,
    action,
    activeTarget,
    displayIconUrl,
    displayDescription,
    availableFrom,
    availableUntil
  } = args

  return { ...args, title: getTokenName(name, padNumberToSameLength(i + 1, count)), index: i, token }
}
```